packages/bento-frontend/src/pages/requestAccess/requestAccessView.js

```diff
@@ -95,8 +95,7 @@ function requestAccessView({ data, classes }) {
   // Init state for inputs.
   const [formValues, setFormValues] = useState(setDefaultValues());
   const [isFormSubmitted, setSubmitted] = useState(false);
-
-  const isInputDisabled = () => isFormSubmitted || (availableArms.length <= 0) || disableSubmit;
+  const isInputDisabled = () => isFormSubmitted || (availableArms.length <= 0) || userStatus === "Disabled";

```

Comment:
@Senait7-ux Can you check is value stored in redux Lower case or Capitalized?

Reply:
Capitalized